paderbox/array/interval/core.py

```diff
@@ -358,6 +358,42 @@ def _intervals_as_str(self):
         i_str = ', '.join(i_str)
         return i_str
 
+    @property
+    def as_serializable(self):
```

Comment:
You could add a type annotation, at least `tuple`, or `Tuple[str, Tuple[int]]`

Reply:
It does not only work with a `tuple`, so I am unsure if it's good to add a type annotation